client/src/util/envUtil.ts

```diff
@@ -51,5 +51,9 @@ export function getWorkbenchLinkValues(): KeyLinkPair[] {
       }
     });
 
+  if (workbenchLinkValues.length === 0) {
```

Comment:
The JS run-time environment throws a pre-defined error if an undefined ENV variable is used. So we don't need to check again.

Reply:
This check was introduced when I thought we needed a prefix as a variable for this function.
If the prefix does not match any env variables, the function will return an empty array. So I would like to keep it for this reason.

Reply:
But seen as this a untestable scenario @SyntaxXeror convinced me we should remove it hehe.